tasmota/tasmota_xdrv_driver/xdrv_76_serial_i2c.ino

```diff
@@ -0,0 +1,163 @@
+/*
+  xdrv_76_serial_i2c.ino - UART to I2C using SC18IM704 compatible protocol
+
+  Copyright (C) 2024  Stephan Hadinger
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifdef USE_I2C_SERIAL
+
+#define XDRV_76           76
+
+class TwoWireSerial;
+
+struct {
+  bool active = false;
+  uint8_t bus;
+  uint8_t tx;
+  uint8_t rx;
+  TwoWireSerial *WireSerial = nullptr;      // replacement object for TwoWire
+} i2c_serial = {0};
+
+class TwoWireSerial : public TwoWire {
+protected:
+  uint8_t tx;
+  uint8_t rx;
+  TasmotaSerial *serial = nullptr;       // serial instance to communicate with SC18IM704
+private:
+public:
+  TwoWireSerial(uint8_t bus_num) : tx(-1), rx(-1), TwoWire(bus_num) {
+    AddLog(LOG_LEVEL_DEBUG_MORE, "ISR: TwoWireSerial(%i)", bus_num);
+  };
```

Comment:
I'd move `= nullptr` to the initializer list:

```c++
TwoWireSerial(uint8_t bus_num) : tx(-1), rx(-1), serial(nullptr), TwoWire(bus_num) {
```

Reply:
Yes indeed. Thx